README.md

```diff
@@ -71,7 +71,7 @@ GO
 Default: `None`
 
 #### `mssql_password`
-The password for the database sa user.
+The password for the database sa user. The password must have minimum length of 8 characters, include uppercase and lowercase letters, base 10 digits or non-alphanumeric symbols.
```

Comment:
What happens if the password does not meet this requirement?

Reply:
Good catch, I will test this properly tomorrow. I remember that if you pass a short password the command will fail saying you the password requirements.

Reply:
The `Set up MS SQL Server` task within the role returns the following error message when I give a simple password `password`:
`fatal: [/home/spetrosi/Downloads/rhel-guest-image-7.9-30.x86_64.qcow2]: FAILED! => {"changed": true, "cmd": ["/opt/mssql/bin/mssql-conf", "-n", "setup", "accept-eula"], "delta": "0:00:00.080953", "end": "2021-02-25 03:46:15.394756", "msg": "non-zero return code", "rc": 1, "start": "2021-02-25 03:46:15.313803", "stderr": "usermod: no changes", "stderr_lines": ["usermod: no changes"], "stdout": "The license terms for this product can be found in\n/usr/share/doc/mssql-server or downloaded from:\nhttps://go.microsoft.com/fwlink/?LinkId=2104294&clcid=0x409\n\nThe privacy statement can be viewed at:\nhttps://go.microsoft.com/fwlink/?LinkId=853010&clcid=0x409\n\nThe specified password does not meet SQL Server password policy requirements because it is not complex enough. The password must be at least 8 characters long and contain characters from three of the following four sets: uppercase letters, lowercase letters, numbers, and symbols.", "stdout_lines": ["The license terms for this product can be found in", "/usr/share/doc/mssql-server or downloaded from:", "https://go.microsoft.com/fwlink/?LinkId=2104294&clcid=0x409", "", "The privacy statement can be viewed at:", "https://go.microsoft.com/fwlink/?LinkId=853010&clcid=0x409", "", "The specified password does not meet SQL Server password policy requirements because it is not complex enough. The password must be at least 8 characters long and contain characters from three of the following four sets: uppercase letters, lowercase letters, numbers, and symbols."]}`
I assume that this is fine that the role fails with such a meaningful error message. I can also print register the output and then print output.stdout_lines for extra visibility. Will that help?